tests/mgbench/benchmark.py

```diff
@@ -686,25 +690,25 @@ def run_target_workload(benchmark_context, workload, bench_queries, vendor_runne
             run_isolated_workload_with_authorization(vendor_runner, client, bench_queries, group, workload, results)
 
 
-# TODO: (andi) Reorder functions in top-down notion in order to improve readibility
+# TODO: (andi) Reorder functions in top-down notion in order to improve readibility -> does this referes to run_target_workloads function or the code around?
 def run_target_workloads(benchmark_context, target_workloads, bench_results):
     for workload, bench_queries in target_workloads:
         log.info(f"Started running {str(workload.NAME)} workload")
-
         benchmark_context.set_active_workload(workload.NAME)
         benchmark_context.set_active_variant(workload.get_variant())
 
+        # TODO(gitbuda): What's the semantic of --export-results-xyz flags? NOTE: avoid nested if/else statements
```

Comment:
@antejavor will the below be correct? In general, what is correct here? 🤔

Reply:
@gitbuda Pokec can be run both `in_memory_transactional`, `on_disk_transactional` and `in_memory_analytical`. 

If you set the flag for `--export-results-on-disk-txn` that mode will be run, by default the `in_memory_transactional` is == `--export-results`. Probably export results needs to become `--export-results-in-memory-transactional`.  

Read this context for flags: 
https://github.com/memgraph/memgraph/blob/master/tests/mgbench/benchmark.py#L119

cc @antepusic